src/open_inwoner/accounts/tests/test_user.py

```diff
@@ -77,6 +77,29 @@ def test_require_necessary_fields_oidc_openinwoner_email(self):
         )
         self.assertTrue(user.require_necessary_fields())
 
+    def test_has_usable_email(self):
+        user_ok1 = UserFactory(email="foo@bar.baz")
+        self.assertTrue(user_ok1.has_usable_email())
+        self.assertTrue(User.is_usable_email("foo@bar.baz"))
```

Comment:
What's the point of testing the class method? The instance method calls the class method, so testing the former should cover the latter, no?